### ISSUE TITLE
Support for MQTT climate hold and away modes has been removed in HA.

### ISSUE THREAD

**Issue by renarena** [zewelor/bt-mqtt-gateway#261]
**Describe the bug**
The homeassistant service set_preset_mode doesn work anymore. 
```
service: climate.set_preset_mode
data:
  preset_mode: comfort
target:
  device_id: xxxxx
```
Error handling message: Entity climate.thermostat_esszimmer_climate does not support this service. 
Logger: homeassistant.components.websocket_api.http.connection
Source: components/websocket_api/connection.py:140
Integration: Home Assistant WebSocket API (documentation, issues)

Seems related to this deprecation info:
> Support for MQTT climate hold and away modes, which was deprecated in Home Assistant Core 2022.3.0, has been removed. The hold and away modes are replaced by preset_mode.

https://github.com/home-assistant/core/pull/66062

**Server (please complete the following information):**
 - Linux
 - Raspbian

**Comment by cybe** [comment]
Hi, could you please try out #266 ?

**Comment by cybe** [comment]
Fixed by #266
